pkg/controllers/hub/membercluster/suite_test.go

```diff
@@ -55,7 +55,7 @@ var _ = BeforeSuite(func() {
 		CRDDirectoryPaths: []string{
 			filepath.Join("..", "..", "..", "..", "config", "crd", "bases"),
 			// The package name must match with the version of the fleet package in use.
-			filepath.Join(build.Default.GOPATH, "pkg", "mod", "go.goms.io", "fleet@v0.11.3", "config", "crd", "bases", "cluster.kubernetes-fleet.io_memberclusters.yaml"),
+			filepath.Join(build.Default.GOPATH, "pkg", "mod", "go.goms.io", "fleet@v0.11.4", "config", "crd", "bases", "cluster.kubernetes-fleet.io_memberclusters.yaml"),
```

Comment:
is there any CRD changes between these two versions?

Reply:
It's related to the cloudConfig parser change in the fleet repo.

Reply:
yeah, related to cloudConfig package. there was a bug in the old version.